health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MobileBezelFrame.js

```
@@ -0,0 +1,125 @@
+    scrollElement.addEventListener("scroll", handleScroll);
+
+    return () => {
+      scrollElement.removeEventListener("scroll", handleScroll);
+      clearTimeout(scrollTimeout);
+    };
+  }, []);
+
+  return (
+    <div className="mobile-bezel-flexContainer">
+      <div className="mobile-bezel-zoom-container">
+        <button className="item zoom-button" onClick={zoomOut}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)